.gitignore

```diff
@@ -1,3 +1,4 @@
 build
 composer.lock
-vendor
\ No newline at end of file
+vendor
+.DS_STORE
```

Comment:
Are you referring to `.DS_STORE`? Because I've already removed it locally. (I'm cleaning this up before tagging the next release.)